liby2/src/genericfrontend.cc

```diff
@@ -657,42 +651,19 @@ main (int argc, char **argv)
 
 // FIXME the whole option parsing sucks **** !
 
-    // Check, if debugger should be enabled
+    // old backward compatibility stuff
```

Comment:
ok, fine for me. I just keep it to be compatible, but if it is not needed, drop is easy.